tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -187,32 +179,11 @@ Then(
   }
 )
 
-When(
-  'the user tries to edit the public link named {string} of file/folder/resource {string} changing following',
-  function (linkName, resource) {
-    return (
-      editPublicLink(linkName, resource)
-        // while editing public link, after clicking the "Save" button, the button should disappear but if it doesn't
-        // we throw "ElementPresentError" error. So, all the error except "ElementPresentError" is caught and thrown back
-        // Also, when no error is thrown, the button seems to disappear, so an error should be thrown in such case as well.
-        .then(() => {
-          throw new Error('ElementPresentError')
-        })
-        .catch((err) => {
-          if (err.message !== 'ElementPresentError') {
-            throw new Error(err)
-          }
-        })
-    )
-  }
-)
```

Comment:
@pascalwengerter is this the step that you were talking about in https://github.com/owncloud/web/pull/6869#discussion_r864758120

The rename one looks okay to me or maybe I'm missing something.

https://github.com/owncloud/web/blob/5af12105117c9866838a1dbce20c57fade5dc208/tests/acceptance/stepDefinitions/publicLinkContext.js#L120-L124